### PR TITLE
Active Scanning and Replacing (AWS) Tokens

### PR DESCRIPTION
**Pull request by speedywizard**

### CODE REVIEW

File: lib/tasks/scraper.rake [hunk 1]
```diff
@@ -0,0 +1,9 @@
+task queue_logs: :environment do
```
Comment (vitalied) [inline lib/tasks/scraper.rake:1]: @speedywizard @AndriiMysko why we need this rake task?
You can use `sidekiq-scheduler` instead

File: app/services/process_logs.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+require 'travis/lock'
+
+class ProcessLogs < ApplicationService
```
Comment (vitalied) [inline app/services/process_logs.rb:3]: @speedywizard @AndriiMysko `ProcessLogsService`

File: app/services/queue_logs.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+require 'travis/lock'
+
+class QueueLogs < ApplicationService
```
Comment (vitalied) [inline app/services/queue_logs.rb:3]: @speedywizard @AndriiMysko `QueueLogsService`

File: spec/factories.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :log do
+    job_id { 1 }
```
Comment (vitalied) [inline spec/factories.rb:3]: @speedywizard @AndriiMysko use Faker::Number

File: spec/factories.rb [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+    content { 'Hello World!' }
```
Comment (vitalied) [inline spec/factories.rb:4]: @speedywizard @AndriiMysko pls put some related content

File: lib/tasks/scraper.rake [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+  loop do
+    Rails.logger.info('Queueing logs...')
+    QueueLogs.new.call
```
Comment (vitalied) [inline lib/tasks/scraper.rake:4]: @speedywizard @AndriiMysko `QueueLogsService.call`

**Review by vitalied — CHANGES_REQUESTED** [review]
@speedywizard @AndriiMysko try to fix rubocop issues

File: spec/factories.rb (hunk 5 above)
Comment (AndriiMysko) [reply spec/factories.rb:4]: Hello @vitalied 
Could you please provide an example of "related content"?

**Comment by AndriiMysko** [comment]
Hello @vitalied 
comments have been fixed.

File: app/services/queue_processing_logs_service.rb [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    rescue => e
+      Rails.logger.error(e.message)
+    end
+  end
+
+  private
+
+  def update_log_entries
+    logs = Log.where(scan_status: :ready_for_scan)
+              .order(id: :desc)
```
Comment (vitalied) [inline app/services/queue_processing_logs_service.rb:18]: @AndriiMysko why order by id desc?

**Pull request closed without merging**